services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -173,7 +174,9 @@ async def get_dynamic_sidecar_status(
     responses={204: {"model": None}},
     summary="stops previously spawned dynamic-sidecar",
 )
+@cancel_on_disconnect
```

Comment:
ok this one I am not sure I get what it has to do with the fix.
This one is useful if the client cuts the communication.

Reply:
Yes, it is an extra to ensure we avoid other possible issues with hanging connection issues.